juriscraper/pacer/attachment_page.py

```diff
@@ -362,26 +364,29 @@ def _get_pacer_doc_id(row):
             doc1_url = url.xpath("./@href")[0]
             return get_pacer_doc_id_from_doc1_url(doc1_url)
 
-    @staticmethod
-    def _get_pacer_seq_no_from_tr(row):
+    def _get_pacer_seq_no_from_tr(self, row):
         """Take a row of the attachment page, and return the sequence number
         from the goDLS function.
         """
```

Comment:
f I remember correctly the `pacer_seq_num` is the same for all the attachments in an entry. Considering this method is called for every row in the attachment page, I think it would give a small performance boost to cache the `pacer_seq_num` as an instance variable (initialised as `None`) if it's already been parsed. We could do something like this:

```suggestion
        """
        if self.pacer_seq_no:
            return self.pacer_seq_no
```

And then store `pacer_seq_no` when it's found:

```
self.pacer_seq_no = go_dls_parts["de_seq_num"]
return self.pacer_seq_no
```

and

```
self.pacer_seq_no = m.group(1)
return self.pacer_seq_no
```

Reply:
I think you're technically correct, however...I think it's best to leave as is as additional validation of that assumption. This attachment parsing code isn't performance critical anyways since attachments are never large enough to cause issues.

Reply:
If anything I should saw we should cache the value and validate each time we parse a row(probably best left to a separate PR).

Reply:
Got it. Yeah that sounds good!